docs/user-guide/configure-certificates-keyring.md

```diff
@@ -0,0 +1,117 @@
+## Configuring Zowe certificates in a key ring	
+
+The JCL member `ZWEKRING` is provided as part of the PDS sample library `SZWESAMP` delivered with Zowe.  The JCL contains the security commands to create the keyring and manage the certificates that Zowe will use.  The JCL must customized before submitting and should be reviewed by a systems programmer familiar with z/OS certificates and keyrings.
+
+After running `ZWEKRING` successfully, the script `zowe-setup-certificates.sh` needs to be run which will create the `KEYSTORE_DIRECTORY` in USS.  Depending on how the `ZWEKRING` member has been customized the configuration file `zowe-setup-certificates.env` that is used to create the `KEYSTORE_DIRECTORY` will need to be customized.
+
+### ZWEKRING
+
+The `ZWEKRING` JCL helps you create a keyring containing a certificate and a local certificate authority that is used to self-sign the certificate.  Other scenarios (such as importing an existing certificate) are described in the `ZWEKRING` JCL with the commands commented out) but are currently unsupported and included in beta mode for early testing and feedback with Zowe 1.15.  If you find any issues please raise an issue at https://github.com/zowe/zowe-install-packaging/issues.  Future releases of Zowe will provide support for more keyring scenarios.  
```

Comment:
This implies to me that scenario 3 is supported and not beta

Reply:
I've added a para to configure-certificates.md to call out that keyrings are beta tech preview,  add the same para to the beginning of configure-certificate-keyring.md, and also change the chapter title to include (Beta Technical Preview) in bold.